public/editor/stateStore.ts

```diff
@@ -16,12 +16,16 @@ const gallerySectionInfos: (Pick<
   (
     | {
         selectionType: "color";
+        colorFormat: "rgba" | "hex";
+        allowAlpha: boolean;
```

Comment:
Is this redundant to have both of these settings? If it's rgba, then it has alpha, that's the a. If it's hex, it can't have alpha. Unless I'm wrong about that, then delete `allowAlpha` and infer it from `colorFormat` alone.

Reply:
Good point, consolidated down to just colorFormat